libs/language-server/workspace/src/lib/read-collections.ts

```diff
@@ -208,7 +208,7 @@ export async function getCollectionInfo(
     const extendedCollections = (
       await Promise.all(
         extendedSchema
-          .filter((extended) => extended !== '@nrwl/workspace')
+          .filter((extended) => extended !== '@nx/workspace')
```

Comment:
won't this line be executed in both v16 and earlier versions? so shouldn't we handle both cases here?